applications/rag/tests/ray_job.py

```diff
@@ -0,0 +1,192 @@
+# This script is copied from the applications/rag/example_notebooks/rag-kaggle-ray-sql-latest.ipynb for testing purposes
```

Comment:
Until this is replaced with notebook exectuion using nbconvert, we need to keep it in sync with the notebook.

Reply:
cc @chiayi